static/app/types/workflowEngine/detectors.tsx

```diff
@@ -24,6 +50,7 @@ interface NewDetector {
 }
 
 export interface Detector extends Readonly<NewDetector> {
+  readonly connectedWorkflows: string[];
```

Comment:
`connectedWorkflows` just got updated to `workflowIds` 😬  sorry about that

also i think this should be under `NewDetector` since we'll want to specify what workflows to connect to when new detectors are created